cookbook/specs/spec_add_side.json

```diff
@@ -0,0 +1,489 @@
+                    "block_parsing": {
+                        "parser_arg": [
+                            "latest"
+                        ],
+                        "parser_func": "DEFAULT"
+                    },
+                    "compute_units": 10,
+                    "enabled": true,
+                    "category": {
+                        "deterministic": true,
+                        "local": false,
+                        "subscription": false,
+                        "stateful": 0
+                    },
+                    "extra_compute_units": 0
+                },
+                {
+                    "name": "side.yield.QueryService/Params",
+                    "block_parsing": {
+                        "parser_arg": [
+                            "latest"
+                        ],
+                        "parser_func": "DEFAULT"
+                    },
+                    "compute_units": 10,
+                    "enabled": true,
+                    "category": {
+                        "deterministic": true,
+                        "local": false,
+                        "subscription": false,
+                        "stateful": 0
+                    },
+                    "extra_compute_units": 0
+                }
+                ],
+                "headers": [],
+                "inheritance_apis": [],
+                "parse_directives": [],
+                "verifications": [
+                    {
+                        "name": "chain-id",
+                        "values": [
+                            {
+                                "expected_value": "S2-testnet-1"
+                            }
+                        ]
+                    },
+                    {
+                        "name": "pruning",
+                        "values": [
+                          {
+                            "latest_distance": 14400
+                        }
+                        ]
+                    }
+                ],
+                "extensions": [
+                    {
+                        "name": "archive",
+                        "cu_multiplier": 5,
+                        "rule": {
+                            "block": 14250
+                        }
+                    }
+                ]
+            },
+              {
+                "enabled": true,
+                "collection_data": {
+                  "api_interface": "tendermintrpc",
+                  "internal_path": "",
+                  "type": "",
+                  "add_on": ""
+                },
+                "apis": [],
+                "headers": [],
+                "inheritance_apis": [],
+                "parse_directives": [],
+                "verifications": [
+                    {
+                        "name": "chain-id",
+                        "values": [
+                            {
+                                "expected_value": "S2-testnet-1"
```

Comment:
I saw that side chain id is "side-testnet-3" in their explorer.
Can you share where you saw this ID? if it's an old one, please update it to "side-testnet-3" (in all api interfaces)

Reply:
Side tesnet 3 is halted they are going to start S2-testnet-1 that's why already named it that way.

Reply:
use the current chain id. If we deploy it tomorrow and providers would want to start giving service, their process will fail on this verification since S2-testnet-1 doesn't exist yet.
When side will start S2-testnet-1, we'll update the spec

Reply:
the S2-testnet-1 started 03.05.2024 so that is the new one

Reply:
ok, keep it as is